docs/t-sql/database-console-commands/dbcc-shrinkdatabase-transact-sql.md

```diff
@@ -168,9 +168,9 @@ Assume you have a couple of log files, a data file, and a database named `mydb`.
 
 For example, if you specify a _target\_percent_ of 25 for shrinking `mydb`, the [!INCLUDE[ssDE](../../includes/ssde-md.md)] calculates the target size for the data file to be 8 MB (6 MB of data plus 2 MB of free space). As such, the [!INCLUDE[ssDE](../../includes/ssde-md.md)] moves any data from the data file's last 2 MB to any free space in the data file's first 8 MB and then shrinks the file.
   
-Assume the data file of `mydb` contains 7 MB of data. Specifying a _target\_percent_ of 30 allows for this data file to be shrunk to the free percentage of 30. However, specifying a _target\_percent_ of 40 doesn't shrink the data file because the [!INCLUDE[ssDE](../../includes/ssde-md.md)] won't shrink a file to a size smaller than the data currently occupies. 
+Assume the data file of `mydb` contains 7 MB of data. Specifying a _target\_percent_ of 30 allows for this data file to be shrunk to the free percentage of 30. However, specifying a _target\_percent_ of 40 doesn't shrink the data file because the [!INCLUDE[ssDE](../../includes/ssde-md.md)] won't shrink a file to a target value greater than the current percentage of free space. 

```

Comment:
Hi @Rob-S, I'm not sure why we're changing the meaning of this sentence. Can you explain?

Other changes look fine.

Reply:
@WilliamDAssafMSFT  Not sure whether or not you saw the comment with that change, so some of this is a repeat.... The "size the data currently occupies" in the example is 7 MB. The example statement is _not_ trying to "shrink [the] file to a size [at or] smaller than 7 MB - it's a lot more than that. Even when specifying 30% free space, it would "shrink [the] file to a size" of 10 MB (7 MB "the data currently occupies" plus 3 MB free space). When specifying 40% free space - a target value greater than the current percentage of free space - that's when "it doesn't shrink the file" - because the "target value is greater than the current percentage of free space."

Reply:
I think the confusion here is whether the intended "target value" is the total size of the data file, not the free space %.  This sentence could be far more clear and thanks for drawing attention to it.

```suggestion
Assume the data file of `mydb` contains 7 MB of data. Specifying a _target\_percent_ of 30 allows for this data file to be shrunk to the free percentage of 30. However, specifying a _target\_percent_ of 40 doesn't shrink the data file because not enough free space can be created in the current total size of the data file.

```